internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -88,7 +96,8 @@ Resources:
     Properties:
       DistributionConfig:
         DefaultCacheBehavior:
-          AllowedMethods: ["GET", "HEAD", "OPTIONS", "PUT", "PATCH", "POST", "DELETE"]
+          Compress: true
+          AllowedMethods: ["GET", "HEAD"]
```

Comment:
Just for learning purpose - what are the considerations behind these changes 👀 ?

Reply:
since the s3 origin is only for static content, the other methods don't make sense - it doesn't make sense to send a `PUT` (for example) to a url that will only ever accept `GET`s. `HEAD` is used by browsers to check if the file has changed (and therefore, should be `GET`'d) without downloading the file.